src/Stages.js

```diff
@@ -11,29 +11,62 @@ class Stages extends Component {
       card: '',
       currentStage: ''
     } 
+    this.scrollStageCard = this.scrollStageCard.bind(this)
 }
 
   componentDidMount(){
     fetch('https://whateverly-datasets.herokuapp.com/api/v1/stages')
       .then(response => response.json())
       .then(stages => {
         this.setState({
-          stages: stages.stages
+          stages: stages.stages.map((stage, i) => {
+            stage.index = i;
+            return stage;
+          })
         })
       })
       .catch(error => console.log(error));
     }
 
     selectStage(e) {
       let stage = this.state.stages.find((stage) => {
-        return stage.name.replace(/\s/g, '') === e.target.classList[0];
+        return stage.index === parseInt(e.target.classList[0]);
       })
       this.setState({
-        currentStage: e.target.classList[0],
+        currentStage: parseInt(e.target.classList[0]),
         card: stage
       })
     }
 
+    setStageIndex() {
+      this.setState({
+        stages: this.state.stages.map((stage, i) => {
+          stage.index = i;
+          return stage;
+        })
+      })
+    }
+
+    scrollStageCard(e) {
+      if (e.target.classList.contains('stage-left-button')) {
```

Comment:
This can be refactored pretty easily.  Pass in a +1 or a -1 to this function depending if the right or left button is clicked.  Then just add the passed in number to index and set the state!